program/rust/src/deserialize.rs

```diff
@@ -16,13 +16,21 @@ use solana_program::program_error::ProgramError;
 
 /// Interpret the bytes in `data` as a value of type `T`
 pub fn load<T: Pod>(data: &[u8]) -> Result<&T, ProgramError> {
-    try_from_bytes(&data[0..size_of::<T>()]).map_err(|_| ProgramError::InvalidArgument)
```

Comment:
nitpick: it would be nice to document the exact error conditions here. I think this errors if there aren't enough bytes to read a T in `data`, but is tolerant of `data` containing more bytes than needed.